server/events_controller_e2e_test.go

```diff
@@ -57,7 +57,7 @@ type LocalConftestCache struct {
 }
 
 func (m *LocalConftestCache) Get(key *version.Version) (string, error) {
-	return exec.LookPath("conftest0.21.0")
+	return exec.LookPath("conftest0.23.0")
```

Comment:
leave all these changes this is probably making the tests fail since the image has an older conftest version. We can update this when we merge to upstream

Reply:
ok, seems like it worked